hooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This file has been added in a different PR

Reply:
yes, and we've just merged that branch to here. So #40 is blocking for this pull.